client/src/legacy/jquery.changetracker.js

```diff
@@ -59,12 +59,18 @@ import debounce from 'lodash.debounce';
 
     this.defaults = {
       fieldSelector: ':input:not(:button,[type="submit"],[type="search"],.gridstate)',
-      ignoreFieldSelector: '.no-change-track,[type="search"]',
+      ignoreFieldSelector: '',
       changedCssClass: 'changed',
     };
 
     var options = $.extend({}, this.defaults, _options);
 
+    // It is mandatory to never allow the search box or markup in the preview to affect change tracking.
+    // Make sure there's no leading comma
+    options.ignoreFieldSelector = `${options.ignoreFieldSelector},.no-change-track,.search-box *,.cms-navigator *`.replace(/^,/, '');
```

Comment:
Adding this here instead of to the default selector list because even if you update the list in a custom section, these should still always be excluded from tracking.